lib/hammer_cli_foreman_remote_execution/job_invocation.rb

```diff
@@ -187,9 +189,14 @@ class RerunCommand < HammerCLIForeman::CreateCommand
     def self.extend_data(invocation)
       if (targeting = invocation['targeting']) && invocation['targeting']['hosts']
         invocation['randomized_ordering'] = targeting['randomized_ordering']
-        if (hosts = targeting['hosts'])
-          invocation['hosts'] = "\n" + hosts.map { |host| " - #{host['name']}" }.join("\n")
+
+        hosts = targeting['hosts'].map do |host|
+          host_item = " - #{host['name']}"
+          host_item << ", Job status: #{host['job_status']}" if host['job_status']
```

Comment:
Not a fan of such approach to show the data, because it can make a little more difficult to easily parse the output in case of csv/yaml output. Since this affects `info` command only (which by default uses `base` output, oriented for human eyes), I don't see a problem, so ACK.